app-emulation/dynamips/dynamips-0.2.21.ebuild

```diff
@@ -36,5 +37,6 @@ src_configure() {
 		-DANY_COMPILER=1
 		-DENABLE_IPV6="$(usex ipv6)"
 	)
+
```

Comment:
You should honor the maintainers style. If they dont want a blank here...

then to other things: You shouldn't use `git mv` but instead make two commits, one for addition one for removal. Also don't rename the patch, it'll be good to know when it was introduced. 

Since @pinkbyte is on devaway, we can take a look, but in future please always make a Bugzilla bug too, because not every dev uses Github at all.

Reply:
>because not every dev uses Github at all.

okay. sry for that

Reply:
It's fine it's fine, even I don't know who use GH and who doesn't. Just saying if you want your contribution to be seen by the maintainer, bugzilla is your best chance. 

As others can't/shouldn't really touch packages maintained by someone else...
(but since @Pinkbyte is currently away, we can process this)